src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/CollectionApiDataType.java

```diff
@@ -64,4 +64,6 @@ public DataType cqlType() {
   public PrimitiveApiDataTypeDef getValueType() {
     return valueType;
   }
+
+  public abstract boolean isFrozen();
```

Comment:
this must have been somewhere else in the code ? 

add comments to explain why we need it here , I think I see why but I also want to check this was not somewhere else in the structur